SECURITY.md

```diff
@@ -15,9 +15,43 @@ Expect a response as fast as possible, within one business day at the latest.
 
 <a name="bounty"></a>
 ## Security Bug Bounties
-We offer bounties for critical security issues. Please see [Bug Bounty
-Compensation](https://forums.solana.com/t/tour-de-sol-updates-to-tour-de-sol-and-bug-bounty-compensation-structure/1132)
-for more details.
+We offer bounties for critical security issues. Please see below for more details.
+
+Loss of Funds:
+$500,000 USD in locked SOL tokens (locked for 12 months)
+* Theft of funds without users signature from any account
+* Theft of funds without users interaction in system, token, stake, vote programs
+* Theft of funds that requires users signature
+
+Consensus/Safety Violations:
+$250,000 USD in locked SOL tokens (locked for 12 months)
+* Consesnus safety violation
+* Tricking a validator to accept an optimistic confirmation or rooted slot without a double vote, etc..
+
+Other Attacks:
+ $100,000 USD in locked SOL tokens (locked for 12 months)
+* Protocol liveness attacks,
+* Eclipse attacks,
+* Remote attacks that partition the network,
+ 
+DoS Attacks:
+ $25,000 USD in locked SOL tokens (locked for 12 months)
+* Non-RPC DoS,
+* Remote resource exaustion via Non-RPC protocols
+ $5,000 USD in locked SOL tokens (locked for 12 months)
```

Comment:
@scriptrunner2049 - this part is a little jumbled up, are these two separate sections?

Reply:
Good pick-up. I've edited the doc.